blockly/blocklyTest.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>BlocklyTest</title>
+  </head>
+  <body>
+    <div id="blocklyDiv" style="height: 480px; width: 600px;"></div>
+    <textarea id="textarea"></textarea>
+    <button id="run">Run</button>
+
+    <script src="../node_modules/blockly/blockly_compressed.js"></script>
+    <script src="../node_modules//blockly/blocks_compressed.js"></script>
+    <script src="../node_modules/blockly/javascript_compressed.js"></script>
+    <script src="../node_modules/blockly/msg/js/en.js"></script>
+    <script src="./custom_blocks/custom.js"></script>
+
+    <xml id="toolbox" style="display: none">
+      <category name="Logic">
+        <block type="controls_if"></block>
+        <block type="controls_repeat_ext"></block>
+        <block type="logic_compare"></block>
+        <block type="math_number"></block>
+        <block type="math_arithmetic"></block>
+        <block type="text"></block>
+        <block type="text_print"></block>
+      </category>
+      <sep></sep>
+      <category name="Movement" colour="210">
+        <block type="move"></block>
+        <block type="rotate"></block>
+      </category>
+    </xml>
+
+    <script>
+      var workspace = Blockly.inject('blocklyDiv',
```

Comment:
wrap the whole blockly editor as a choo component. Use lifecycle hooks for initialization